### PR TITLE
tidb-scheduler: improve some expressions

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? (Required)

improve some expressions in tidb-scheduler doc

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated in: https://github.com/pingcap/docs/pull/2062/
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/2228

### CODE REVIEW

File: tidb-in-kubernetes/reference/components/tidb-scheduler.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 
-TiDB 集群核心包括 PD，TiKV 以及 TiDB 三个组件，每个组件又都是由多个节点组成，PD 是一个 Raft 集群，TiKV 是一个多 Group Raft 集群，并且这两个组件又都是有状态的。因此，默认 K8s 的调度器的调度规则已经无法满足 TiDB 集群的调度需求，需要扩展 K8s 的调度规则，目前，TiDB Operator 实现了如下几种自定义的调度规则：
+TiDB 集群包括 PD，TiKV 以及 TiDB 三个核心组件，每个组件又是由多个节点组成，PD 是一个 Raft 集群，TiKV 是一个多 Group Raft 集群，并且这两个组件都是有状态的。因此，默认 Kubernetes 的调度器的调度规则已经无法满足 TiDB 集群的调度需求，需要扩展 Kubernetes 的调度规则。
+
+目前，TiDB Scheduler 实现了如下几种自定义的调度规则。
 
 ### PD 组件
 
-确保每个 Node 节点上调度的 PD 个数小于 `Replicas / 2`，例如：
+调度规则一：确保每个节点上调度的 PD 副本个数小于 `Replicas / 2`。例如：
```
Comment (yikeke) [inline tidb-in-kubernetes/reference/components/tidb-scheduler.md]: 这里写“PD 副本”还是“PD 实例”比较好？ @weekface

File: tidb-in-kubernetes/reference/components/tidb-scheduler.md [hunk 2]
```diff
@@ -40,21 +42,24 @@ TiDB 集群核心包括 PD，TiKV 以及 TiDB 三个组件，每个组件又都
 
 ### TiDB 组件
 
-实现了稳定调度：在 TiDB 组件滚动更新的时候，尽量将其调度回原来的节点，这对于手动将 Node IP + NodePort 挂载在 LB 后端的场景比较有帮助，避免升级集群后 Node IP 发生变更需要重新调整 LB，这样可以减少滚动更新时对集群的影响。
+调度规则三：在 TiDB 组件滚动更新的时候，尽量将其调度回原来的节点。
```
Comment (yikeke) [inline tidb-in-kubernetes/reference/components/tidb-scheduler.md]:
```suggestion
调度规则三：在 TiDB 实例滚动更新的时候，尽量将其调度回原来的节点。
```
“其”表示具体的东西比较好，所以把“TiDB 组件”改成“TiDB 实例”或者“TiDB 副本”。“实例”还是“副本”哪个准确？ @weekface

File: tidb-in-kubernetes/reference/components/tidb-scheduler.md (hunk 1 above)
Comment (weekface) [reply tidb-in-kubernetes/reference/components/tidb-scheduler.md]: 用 “PD 实例” 吧。

File: tidb-in-kubernetes/reference/components/tidb-scheduler.md (hunk 2 above)
Comment (weekface) [reply tidb-in-kubernetes/reference/components/tidb-scheduler.md]: 用 “实例”

File: tidb-in-kubernetes/reference/components/tidb-scheduler.md (hunk 1 above)
Comment (yikeke) [reply tidb-in-kubernetes/reference/components/tidb-scheduler.md]: ok，已改

File: tidb-in-kubernetes/reference/components/tidb-scheduler.md (hunk 2 above)
Comment (yikeke) [reply tidb-in-kubernetes/reference/components/tidb-scheduler.md]: ok，已改

**Pull request merged**